diff --git a/dist/content/loaders/file.js b/dist/content/loaders/file.js
index b9ced5a..8a450d4 100644
--- a/dist/content/loaders/file.js
+++ b/dist/content/loaders/file.js
@@ -1,6 +1,6 @@
 import { existsSync, promises as fs } from "node:fs";
 import { fileURLToPath } from "node:url";
-import yaml from "js-yaml";
+import * as yaml from "js-yaml";
 import * as toml from "smol-toml";
 import { FileGlobNotSupported, FileParserNotFound } from "../../core/errors/errors-data.js";
 import { AstroError } from "../../core/errors/index.js";
diff --git a/dist/core/config/settings.js b/dist/core/config/settings.js
index 42cbff5..0cc40f7 100644
--- a/dist/core/config/settings.js
+++ b/dist/core/config/settings.js
@@ -1,6 +1,6 @@
 import path from "node:path";
 import { fileURLToPath, pathToFileURL } from "node:url";
-import yaml from "js-yaml";
+import * as yaml from "js-yaml";
 import * as toml from "smol-toml";
 import { getContentPaths } from "../../content/index.js";
 import createPreferences from "../../preferences/index.js";
